src/test/java/com/amadeus/jenkins/plugins/workflow/libs/HttpRetrieverIntegrationTests.java

```diff
@@ -225,18 +230,23 @@ FilePath buildJobWithLibrary(@Nullable String fixtureName, @Nullable String libr
                             .willReturn(WireMock.notFound()));
         }
         globalLibraries.getLibraries().add(new LibraryConfiguration(
-                libraryName,
-                new HttpRetriever(urlBuilder.apply(libraryName), credentials.getId(), withPreemptiveAuth)
+                "foo",
+                new HttpRetriever(urlBuilder.apply("foo"), credentials.getId(), withPreemptiveAuth)
         ));
         WorkflowJob p = j.jenkins.createProject(WorkflowJob.class, "p");
         p.setDefinition(new CpsFlowDefinition(importScript, true));
 
         WorkflowRun run = j.buildAndAssertSuccess(p);
-        return new FilePath(run.getRootDir().toPath().resolve("libs").resolve(libraryName).toFile());
+        List<Path> children = Files.list(run.getRootDir().toPath().resolve("libs"))
+                .filter(Files::isDirectory)
+                .collect(Collectors.toList());
+        assertThat(children, hasSize(1));
+        return children.get(0);
     }
 
     @Test
     public void retrievesWithPreemptiveAuthIfNeeded() throws Exception {
+        Assume.assumeFalse("TODO flakes on Windows due to open file handles", Functions.isWindows());
```

Comment:
```suggestion
        Assume.assumeFalse("TODO fails on Windows", Functions.isWindows());
```
https://github.com/jenkinsci/jenkins-test-harness/pull/654